src/main/java/uk/gov/hmcts/darts/task/api/AutomatedTaskName.java

```diff
@@ -36,7 +36,7 @@ public enum AutomatedTaskName {
     CASE_EXPIRY_DELETION_TASK_NAME("CaseExpiryDeletion", "${darts.automated.task.case-expiry-deletion.enabled:false}"),
     ASSOCIATED_OBJECT_DATA_EXPIRY_DELETION_TASK_NAME("AssociatedObjectDataExpiryDeletion"),
     MANUAL_DELETION("ManualDeletion", "${darts.manual-deletion.enabled:false}"),
-    PROCESS_ARM_RPO_PENDING("ProcessARMRPOPending", "!${darts.automated.task.process-e2e-arm-rpo:false}"),
+    PROCESS_ARM_RPO_PENDING("ProcessARMRPOPending", "!${darts.automated.task.process-e2e-arm-rpo-pending.process-e2e-arm-rpo:false}"),
     AUDIO_LINKING_TASK_NAME("AudioLinking"),
     PROCESS_E2E_ARM_PENDING_TASK_NAME("ProcessE2EArmRpoPending"),
```

Comment:
Shouldn't this one use the feature flag used on line 39 as well?

Reply:
This is un changed code but the PROCESS_E2E_ARM_PENDING_TASK_NAME should use the inverse flag so will update